nextjs/src/lib/map/useExplorer.ts

```diff
@@ -109,9 +111,9 @@ export function useExplorer() {
   }
 
   function show() {
-    if (isValidEntity(state)) {
-      return setState((s) => ({ showExplorer: true }))
-    }
+    // if (isValidEntity(state)) {
```

Comment:
question: is this not needed anymore?

Reply:
Yes, because the area explorer can now show UK-wide stuff without an entity.